server/playground.py

```diff
@@ -473,16 +503,28 @@ async def handle_postgres_backend_request():
 
             )
             query_results_page = await query_results_page_exec.fetchall()
-            query_results_columns = query_results_page_exec.description
+
+        # Provide types for the columns
+        async with postgresql_connexion.cursor() as cur:
+            # oid of columns of query results are provided in the "description" attribute
```

Comment:
oid refers to object identifiers ? https://www.postgresql.org/docs/current/datatype-oid.html